src/overlays/gamestates/ovl_first_game/sys_romcheck.c

```diff
@@ -0,0 +1,20 @@
+#include "first_game.h"
+#include "carthandle.h"
+#include "m_nmi_buf.h"
+#include "attributes.h"
+
+void sys_romcheck(void) {
+    u32 sp24;
+    u8 version;
+    UNUSED s32 pad;
+
+    // Read the last 4 bytes from the ROM header
+    osEPiReadIo(carthandle, 0x3C, &sp24);
+
+    version = sp24;
+    sp24 = 0x23;
```

Comment:
In GC, they check for a specific Version ID number that is related to debug, this could be a define like.. 
`DEBUG_ROM_VERSION`? Since 0x3F is the version number in headers (source: en64)

Reply:
I'll check into this by changing the header and seeing what I get